lib/fs.ts

```diff
@@ -80,6 +80,18 @@ class FSPath {
 		return fs.readFileSync(this.path, options) as Buffer;
 	}
 
+	checkIfExists(): Promise<boolean> {
```

Comment:
No other function prepends `checkIf` to its name.

```suggestion
	exists(): Promise<boolean> {
```

Reply:
Also do we need this? In theory, `isFile`, `isFolder`, etc should cover this use-case.

Reply:
> Also do we need this? In theory, isFile, isFolder, etc should cover this use-case.

I guess I missed them when skimming through it, but `FS#exists` maybe be worth keeping in other cases where we cannot infer if the file is a file or directory.

Reply:
This hasn't been changed yet...